host-interaction/gui/window/get-enumeration-functions.yml

```diff
@@ -0,0 +1,19 @@
+rule:
+  meta:
+    name: get enumeration functions
+    namespace: host-interaction/gui/window
```

Comment:
since some of the APIs are not window-centric, lets relax the namespace to `host-interaction/gui`